lib/cucumber/core/test/result.rb

```diff
@@ -5,13 +5,19 @@ module Core
     module Test
       module Result
 
+        # Defines a status reader on a result class for the given status
+        #
         # Defines predicate methods on a result class with only the given one
         # returning true
-        def self.status_queries(status)
+        def self.status_methods(status)
           Module.new do
+            define_method :status do
```

Comment:
Maybe I'm being picky, but I'm inclined to call this `:to_sym`. It doesn't sound right to me for a _result_ to _have a status_. Do you know what I mean?